src/models/appointment.ts

```diff
@@ -0,0 +1,21 @@
+import { DataTypes } from 'sequelize';
+import { AppointmentsAttributes } from '../types/index';
```

Comment:
no need to  type index
```suggestion
import { AppointmentsAttributes } from '../types';
```